samples/update_connection.py

```diff
@@ -0,0 +1,63 @@
+####
+# This script demonstrates how to update a connections credentials on a server to embed the credentials
+#
+# To run the script, you must have installed Python 2.7.X or 3.3 and later.
+####
+
+import argparse
+import getpass
+import logging
+
+import tableauserverclient as TSC
+
+
+def main():
+    parser = argparse.ArgumentParser(description='Get all of the refresh tasks available on a server')
```

Comment:
The description is for the wrong sample (I bet a lot of our samples are, actually)